install_cli.php

```diff
@@ -224,16 +224,14 @@
     $dTool = new \Sugarcrm\Tidbit\DataTool($storageType);
     $dTool->fields = $bean->field_defs;
 
-    $dTool->table_name = $bean->table_name;
+    $dTool->table_name = $bean->getTableName();
     $dTool->module = $module;
 
     if (!empty($GLOBALS['as_populate']) && $activityGenerator->willGenerateActivity($bean)) {
         echo "\n\tWill create " . $activityGenerator->calculateActivitiesToCreate($total) . " activity records";
     }
     echo "\n\tHitting DB... ";
 
-    $beanInsertBuffer = new \Sugarcrm\Tidbit\InsertBuffer($dTool->table_name, $storageAdapter, $insertBatchSize);
```

Comment:
We have to stay insertBuffer creation there, cause otherwise \Sugarcrm\Tidbit\InsertBuffer::destruct will be called after each bean generation and stored in DB as only once record per SQL Insert statement. 
By default we "flush" buffer (insert into DB) every 20 records but on large data set, we use 1k or 5k records per one insert, that is really speed up data generation.

Same for custom fields check: we can set $useCstmTable and create Custom buffer there and addInstallData later